Modules/clinic/signalmodule.c.h

```diff
@@ -168,7 +168,7 @@ signal_siginterrupt(PyObject *module, PyObject **args, Py_ssize_t nargs)
 #if defined(HAVE_SETITIMER)
 
 PyDoc_STRVAR(signal_setitimer__doc__,
-"setitimer($module, which, seconds, interval=0.0, /)\n"
+"setitimer($module, which, seconds, interval=None, /)\n"
```

Comment:
I don't see any point in changing the default to None.

Reply:
Oops, it wasn't my intent. I fixed it.